app/routes/Account/components/LockingState/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import CryptoJs from 'crypto-js'
+
+import ClearFix from 'components/ClearFix'
+import styles from './index.css'
+import actions from './actions'
+
+class LockingState extends Component {
+  constructor(props) {
+    super(props)
+
+    this.handleUnlockAccount(props)
+  }
+
+  handleLockAccount = () => {
+    this.props.onLockAccount()
+  }
+
+  handleUnlockAccount = (props) => {
+    const hasPassword = props && props.properties && props.properties.state
+    if (!hasPassword)
+      return
+
+    const { password } = props.properties.state
+    const unlockingTime = new Date()
+    const encryptedMnemonic = props.account.secondFA.seed
+    const mnemonic = CryptoJs.AES.decrypt(
+      encryptedMnemonic,
+      password,
+    ).toString(CryptoJs.enc.Utf8)
+
+    this.props.onUnlockAccount(mnemonic, unlockingTime.toISOString())
+  }
+
+  render() {
+    const { account } = this.props
+
+    return (
+      <div className={styles.lockedState}>
+        {(account.lockedState)
+          ?
+          <div className={styles.lockedAccounts}>
+            <div className={styles.state}>Safes are locked</div>
+            <Link to={{
+              pathname: '/password',
+              state: {
```

Comment:
What it does ```state: dest: '/account'```?

Reply:
"dest" param is used to tell the Password component where to go after finishing. This way the Password component can redirect to a different page depending on the screen flow where it is used.